airbyte-integrations/connectors/source-google-analytics-data-api/source_google_analytics_data_api/source.py

```diff
@@ -103,6 +100,10 @@ class GoogleAnalyticsDataApiBaseStream(GoogleAnalyticsDataApiAbstractStream):
     https://developers.google.com/analytics/devguides/reporting/data/v1/rest/v1beta/properties/runReport
     """
 
+    # set page_size to 100000 due to determination of maximum limit value in official documentation
+    # https://developers.google.com/analytics/devguides/reporting/data/v1/basics#pagination
+    PAGE_SIZE = 100000
```

Comment:
Since the page_size can is now being modified by the check method, it doesn't really make sense to present PAGE_SIZE as a constant anymore. We should just make it a regular variable that defaults to 100000, but has a first class method to set it to the 100 for check

Reply:
done